Tests/js/views/assets/ez-contenteditview-tests.js

```diff
@@ -100,7 +101,7 @@ YUI.add('ez-contenteditview-tests', function (Y) {
 
             this.view.template = function (variables) {
                 Y.Assert.isObject(variables, "The template should receive some variables");
-                Y.Assert.areEqual(6, Y.Object.keys(variables).length, "The template should receive 6 variables");
+                Y.Assert.areEqual(7, Y.Object.keys(variables).length, "The template should receive 7 variables");
```

Comment:
this is not valid any more, did you forget to run the unit test before pushing ? :)